array/blit_js.mbt

```diff
@@ -25,7 +25,7 @@
 ///
 /// Example:
 ///
-/// ```moonbit
+/// ```mbt check
 ///   let original = [1, 2, 3]
```

Comment:
this will be skipped by the toolchain -> `mbt test`
cc @bzy-debug we should emit a warning for this unexpected missing